scripts/app/test_internal_chain.py

```diff
@@ -22,19 +23,20 @@ def test(
         self,
         network: Network,
         offset: Offset,
-        limit: Limit
+        limit: Limit,
+        delay: Delay
     ):
         # mainnet headless URL and initial setup
         mainnet_headless = "odin-full-state.nine-chronicles.com"
         internal_target_validator = "odin-internal-validator-5.nine-chronicles.com"
-        sleep_time = 5
+        sleep_time = delay
```

Comment:
just curious is it intended default sleep time reduced in odin-internal

Reply:
Starting from this update, default sleep time is initially set to 0 for both networks(https://github.com/planetarium/9c-infra/pull/1511/files#diff-18621719fb04b6b3d0276ad0cffe51f2f8fd9bc3de1e51bb93ca00c6bd473349R22) and you can customize the interval upon each test.